### PR TITLE
doc: clarifies opm usage and what tool can help with bundles

### PR DESCRIPTION
**Pull request by camilamacedo86**
**Description of the change:**
clarifies opm usage and what tool can help with bundles

**Motivation for the change:**
Closes: https://github.com/operator-framework/operator-registry/issues/562

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -63,6 +64,8 @@ podman push quay.io/my-container-registry-namespace/my-manifest-bundle:latest
 
 Of course, this build step can be done with any other OCI spec container tools like `docker`, `buildah`, `libpod`, etc.
 
+Note that you do not create your bundle manually. [Operator-SDK](https://github.com/operator-framework/operator-sdk) provide features and helpers to build, to update, to validate and to test bundles for projects which follows the SDK layout or not. For more information check its documentations over [Integration with OLM](https://sdk.operatorframework.io/docs/olm-integration)
```
Comment (dinhxuanvu) [inline README.md:67]: I would remove the first sentence `Note that you do not create your bundle manually.`
This is an upstream project so we shouldn't dictate what users can or can't do manually on their own. Simply pointing out that SDK provides tooling and features to build, to update, to validate and to test bundles is sufficient.

Comment (camilamacedo86) [reply README.md:67]: It is missing a word actually : note that you do not need to create the ....

Comment (camilamacedo86) [inline README.md:67]:
```suggestion
Note that you do not need to create your bundle manually. [Operator-SDK](https://github.com/operator-framework/operator-sdk) provide features and helpers to build, to update, to validate and to test bundles for projects which follows the SDK layout or not. For more information check its documentations over [Integration with OLM](https://sdk.operatorframework.io/docs/olm-integration)
```

Comment (camilamacedo86) [reply README.md:67]: @dinhxuanvu 

is it making more sense now?

File: README.md [hunk 2]
```diff
@@ -63,6 +64,8 @@ podman push quay.io/my-container-registry-namespace/my-manifest-bundle:latest
 
 Of course, this build step can be done with any other OCI spec container tools like `docker`, `buildah`, `libpod`, etc.
 
+Note that you do not need to create your bundle manually. [Operator-SDK](https://github.com/operator-framework/operator-sdk) provide features and helpers to build, to update, to validate and to test bundles for projects which follows the SDK layout or not. For more information check its documentations over [Integration with OLM](https://sdk.operatorframework.io/docs/olm-integration)
```
Comment (camilamacedo86) [inline README.md:67]: @dinhxuanvu

is it making more sense now?

Comment (dinhxuanvu) [reply README.md:67]: That does sound better.

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**